wallet-services/src/main/resources/locale/addon/Wallet_en.properties

```diff
@@ -424,7 +424,7 @@ exoplatform.wallet.button.requestAuthorization=Request authorization
 exoplatform.wallet.button.manageKeys=Manage Keys
 exoplatform.wallet.button.cancel=Cancel
 exoplatform.wallet.button.send=Send
-exoplatform.wallet.button.sendfunds=Send funds
+exoplatform.wallet.button.sendfunds=Send Funds
```

Comment:
Use CSS for this https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Reply:
done !